libraries/AP_BattMonitor/AP_BattMonitor_UAVCAN.cpp

```diff
@@ -55,6 +59,14 @@ void AP_BattMonitor_UAVCAN::subscribe_msgs(AP_UAVCAN* ap_uavcan)
         AP_BoardConfig::allocation_error("UAVCAN BatteryInfoAux subscriber start problem");
         return;
     }
+    
+    uavcan::Subscriber<mppt::Stream, MpptStreamCb> *mppt_stream_listener;
+    mppt_stream_listener = new uavcan::Subscriber<mppt::Stream, MpptStreamCb>(*node);
+    const int mppt_stream_listener_res = mppt_stream_listener->start(MpptStreamCb(ap_uavcan, &handle_mppt_stream_trampoline));
+    if (mppt_stream_listener_res < 0) {
+        AP_HAL::panic("UAVCAN Mppt::Stream subscriber start problem\n\r");
```

Comment:
Surely we don't want a panic here do we?

Reply:
Good catch! That's as copy paste from an older branch where we were doing it that way here. This new way looks much better.

Reply:
I fixed this and added it to your commit since you're the one who caught it